hat/assets/js/apps/Iaso/domains/instances/components/InstanceDetailsLocation.tsx

```diff
@@ -0,0 +1,150 @@
+                        }
+                        valueTitle={
+                            <OrgUnitLabel orgUnit={ou} withType={false} />
+                        }
+                        value={<OrgUnitDisplay orgUnit={ou} withType={false} />}
+                    />
+                ))}
+                {currentInstance.org_unit && (
+                    <InstanceDetailsField
+                        label={formatMessage(MESSAGES.source_ref)}
+                        valueTitle={
+                            <OrgUnitLabel
+                                orgUnit={currentInstance.org_unit}
+                                withType={false}
+                            />
+                        }
+                        value={
+                            <OrgUnitSourceRefDisplay
+                                orgUnit={currentInstance.org_unit}
+                            />
+                        }
+                    />
+                )}
+                {currentInstance.org_unit &&
+                    currentInstance.org_unit.groups && (
+                        <InstanceDetailsField
+                            label={formatMessage(MESSAGES.groups)}
+                            value={
+                                currentInstance.org_unit.groups.length > 0
+                                    ? currentInstance.org_unit.groups
+                                          .map(g => g.name)
+                                          .join(', ')
+                                    : null
+                            }
+                        />
+                    )}
+                {!currentInstance.latitude &&
+                    !currentInstance.longitude &&
+                    currentInstance.org_unit &&
+                    currentInstance.org_unit.latitude &&
+                    currentInstance.org_unit.longitude && (
+                        <>
+                            <InstanceDetailsField
+                                label={formatMessage(MESSAGES.latitude)}
+                                value={currentInstance.org_unit.latitude}
+                            />
+                            <InstanceDetailsField
+                                label={formatMessage(MESSAGES.longitude)}
+                                value={currentInstance.org_unit.longitude}
+                            />
+                        </>
+                    )}
+                {currentInstance.latitude && currentInstance.longitude && (
+                    <>
+                        <InstanceDetailsField
+                            label={formatMessage(MESSAGES.latitude)}
+                            value={currentInstance.latitude}
+                        />
+                        <InstanceDetailsField
+                            label={formatMessage(MESSAGES.longitude)}
+                            value={currentInstance.longitude}
+                        />
+                    </>
+                )}
```

Comment:
You can declare variable at the beginning  avoiding to repeat `currentInstance`:
`const {longitude, atitude, precision, ...} = currentInstance`

It will make it more readable too